conjur/init/init_controller.py

```diff
@@ -45,24 +45,56 @@ def load(self):
         self.write_conjurrc()
 
         sys.stdout.write("Configuration initialized successfully!\n")
-        sys.stdout.write("To begin using the CLI, log in to the Conjur server by" \
-                         "running `conjur authn login`")
+        sys.stdout.write("To begin using the CLI, log in to the Conjur server by " \
+                         "running `conjur login`\n")
 
     def get_server_certificate(self):
         """
         Method to get the certificate from the Conjur endpoint detailed by the user
         """
         # pylint: disable=line-too-long
         if self.conjurrc_data.appliance_url is None:
-            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ")
+            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ").strip()
+            if self.conjurrc_data.appliance_url is None or self.conjurrc_data.appliance_url == '':
+                # pylint: disable=raise-missing-from
+                raise RuntimeError("Error: URL is required\n")
+
 
         url = urlparse(self.conjurrc_data.appliance_url)
+        # TODO: Factor out the following URL validation to ConjurrcData class
+        # pylint: disable=pointless-string-statement
+
+        # This condition catches if a user provides input <hostname>
+        # without HTTP/HTTPS prefix. If so, we default the behavior
+        # to HTTPS because this is what we currently support
```

Comment:
This is not the correct place to do these validations.
Create a method `get_conjur_url`, and then validate the input there.

Reply:
added a todo